### PR TITLE
Update cores list

### PR DESCRIPTION
**Pull request by hughbreslin**
Added Microchip MIV IMC and updated PF SoC with Icicle Kit release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,5 +122,5 @@ GAP8 | GreenWaves Technologies | [Product page](https://greenwaves-technologies.
...
+PolarFire SoC | Microchip | [Product Page](https://www.microsemi.com/product-directory/soc-fpgas/5498-polarfire-soc-fpga), [IDE with Renode platform](https://www.microsemi.com/product-directory/design-tools/4879-softconsole) | U54 (4 cores), E51 (management core) | RV64GC(U54), RV64IMAC(E51) | Linux | [Microchip Icicle Kit](https://www.microsemi.com/existing-parts/parts/152514), [HiFive Unleashed Expansion Board](https://www.microsemi.com/hifive-unleashed-expansion-board) | Icicle Kit ES available Q3 2020, HiFive Unleased Expansion Board - Q2 2018
 GD32VF103 | GigaDevice | [Product listing](https://www.gigadevice.com/products/product-finder/?fwp_processor_type=risc-v&fwp_microcontrollers_product_line=mainstream&fwp_microcontrollers_product_series=gd32vf103),[Datasheets](https://github.com/riscv-mcu/GD32VF103_DataSheets) | [Bumblebee Core](http://dl.sipeed.com/LONGAN/Nano/DOC/Bumblebee%20core%20datasheet_en.pdf) | RV32IMAC | RTOS | GD32VF103V-EVAL, [Longan Nano](http://longan.sipeed.com/) etc.| Public since 2019Q4
```
Comment (public-UCTECHIP) [inline README.md:126]: WH32(32bit) | UC Techip | https://www.uctechip.com/#product | 1.10 | RV32GCX | Chisel | UC Techip  Commercial  License

**Review by public-UCTECHIP — APPROVED** [review]
WH32(32bit) | UC Techip | https://www.uctechip.com/#product | 1.10 | RV32GCX | Chisel | UC Techip Commercial License

**Review by public-UCTECHIP — APPROVED** [review]
WH32(32bit) | UC Techip | [Website](https://www.uctechip.com/#product) | 1.10 | RV32GCX | Chisel | UC Techip Commercial License

**Comment by mgielda** [comment]
Hi Hugh, thanks for the contribution, a pleasure working with you as always!

**Comment by mgielda** [comment]
(I think the UCTECHIP edits were erroneous)

**Pull request merged**